bubblegum/program/README.md

```diff
@@ -1,4 +1,436 @@
-# Bubblegum - NFTs in Merkle Trees
-Supports decompressing bubblegum NFTs into either Tokenkeg tokens or Token22 tokens. The benefit of decompressing a bubblegum 
-NFT is that normal tokens can be moved into a custodial wallet and freely transferred without relying on RPC nodes to serve 
-your NFT data from an off-chain database.
+# Bubblegum - Compressed Metaplex NFTs
+
+> ⚠️ **Bubblegum is currently experimental and has not been formally audited. Use in production
+> at your own risk.**
+
+## Overview
+
+`Bubblegum` is the Metaplex Protocol program for creating and interacting with compressed Metaplex NFTs.  With Bubblegum you can:
+* Create a tree
```

Comment:
You mention `tree` in the overview but at this point the concept of a "tree" is not defined. Perhaps we should have a single sentence to say that a Merkle tree is used to validate the state of compressed NFTs (or something like that) in the paragraph above.

Reply:
Done